### ISSUE TITLE
A Hybrid Approach for 6DoF Pose Estimation. (arXiv:2011.05669v1 [cs.CV])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#813]
A Hybrid Approach for 6DoF Pose Estimation. (arXiv:2011.05669v1 [cs.CV])<br>
https://ift.tt/3eYjz88<br>
<p>We propose a method for 6DoF pose estimation of rigid objects that uses a state-of-the-art deep learning based instance detector to segment object instances in an RGB image, followed by a point-pair based voting method to recover the object's pose. We additionally use an automatic method selection that chooses the instance detector and the training set as that with the highest performance on the validation set. This hybrid approach leverages the best of learning and classic approaches, using CNNs to filter highly unstructured data and cut through the clutter, and a local geometric approach with proven convergence for robust pose estimation. The method is evaluated on the BOP core datasets where it significantly exceeds the baseline method and is the best fast method in the BOP 2020 Challenge.</p>
<br>
<br>
via cs.CV updates on arXiv.org http://arxiv.org/<br>
